fastlane/Fastfile

```diff
@@ -10,6 +11,10 @@ files_to_update_phc_version = {
     'android/build.gradle' => ['com.revenuecat.purchases:purchases-hybrid-common:{x}'],
     'package.json' => ['"@revenuecat/purchases-typescript-internal-esm": "{x}"']
 }
+files_to_update_phc_ui_version = {
```

Comment:
Hmm so since PHC and PHC_UI always match the same version (at least for now), I think we could potentially just add these files to the above `files_to_update_phc_version`? That's how it works for our existing hybrids that support the UI version: https://github.com/RevenueCat/purchases-flutter/blob/ed61c5614884dbe40fcbe48ab0ca8379399cd04d/fastlane/Fastfile#L163-L176